sdk/servicebus/service-bus/test/internal/batchReceiver.spec.ts

```diff
@@ -1070,60 +1071,62 @@ describe("Batching Receiver", () => {
     });
 
     describe("Session Batch Receiver - disconnects (sessions)", function(): void {
-      let serviceBusClient: ServiceBusClientForTests;
-      let sender: ServiceBusSender;
-      let receiver: ServiceBusSessionReceiver;
+      let serviceBusClientInner: ServiceBusClientForTests;
+      let senderInner: ServiceBusSender;
+      let receiverInner: ServiceBusSessionReceiver;
```

Comment:
The reason this test block is redefining the sender and receiver as we want to target session enabled queue/topic, but there should not be a reason to have a different serviceBusClient variable. I would recommend trying
- rename `serviceBusClientInner` back to `serviceBusClient` and remove the variable declaration here. 
- rename `senderInner` and `receiverInner` to `sessionSender` and `sessionReceiver` respectively